src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/PartiesNotified.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.model.single.hearing;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PartiesNotified {
+
+    private Long hearingID;
```

Comment:
@MohammedHMCTS this variable type for hearingID is **String**
HMC PR fyi https://github.com/hmcts/hmc-cft-hearing-service/pull/193/files#diff-a6cb9f13bf2b4ae9b10fae6e8c5db14121cf1905b6bdfd1092d3d0b5dcfb4e81R12

Reply:
Good spot I updated that, I was aware of the change to String but forgot to update it... Sort now :)